src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -481,11 +487,8 @@ private static ITaskSeriesTimer StartParameterLogTimer(IRecurrentCommand updateC
 
                 try
                 {
-                    using (logger.BeginScope(new Dictionary<string, object>
-                    {
-                        [LogConstants.CategoryNameKey] = LogCategories.CreateFunctionCategory(instance.FunctionDescriptor.LogName),
-                        [LogConstants.LogLevelKey] = LogLevel.Information
-                    }))
+                    using (logger.BeginScope(new ReadOnlyScopeDictionary(ExecuteWatcherScopeKeys, 
+                        new object[] {LogCategories.CreateFunctionCategory(instance.FunctionDescriptor.LogName), LogLevel.Information})))
```

Comment:
Side note for a future improvement (@mathewc) -- would caching these category strings would give us a boost?

Reply:
Maybe we could push it to the `.FunctionDescriptor` and have a property in there?